packages/wrangler/src/pages/publish.tsx

```diff
@@ -318,14 +318,17 @@ export const Handler = async ({
 
 	if (_headers) {
 		formData.append("_headers", new File([_headers], "_headers"));
+		logger.log(`✨ Uploading _headers`);
 	}
 
 	if (_redirects) {
 		formData.append("_redirects", new File([_redirects], "_redirects"));
+		logger.log(`✨ Uploading _redirects`);
 	}
 
 	if (builtFunctions) {
 		formData.append("_worker.js", new File([builtFunctions], "_worker.js"));
+		logger.log(`✨ Uploading Pages Functions`);
```

Comment:
```suggestion
		logger.log(`✨ Uploading Functions`);
```
I'd probably just remove Pages here. We don't have it anywhere else